cardano-cli/test/cli/stake-address-deregistration-certificate/run

```diff
@@ -18,6 +18,6 @@ ${CARDANO_CLI} shelley stake-address deregistration-certificate  \
 fail_on_error $?
 
 assert_file_exists "${TEST_DIR}/deregistration.cert"
-assert_string 'DelegationCertificateShelley' "${TEST_DIR}/deregistration.cert"
+assert_string 'Stake Address Deregistration Certificate' "${TEST_DIR}/deregistration.cert"
```

Comment:
Note that "Stake Address Deregistration Certificate" is the `TextViewTitle`, not the `TextViewType` (as "DelegationCertificateShelley" was). The new `TextViewType` is: "Certificate".

Reply:
good point

Reply:
I'm moving this test to hedgehog aswell.